components/user/UserLink.js

```diff
@@ -15,7 +17,7 @@ export default function UserLink({ link, username, displayStatsPublic }) {
       // TODO: link not found
       console.log("ERROR link not found ", e);
     }
-    // TODO: 2. visit link
+    router.push(`${link.url}`)
```

Comment:
Thank you 👍 

I think it can be the variable without the backticks

```suggestion
    router.push(link.url)
```

Reply:
@hemantwasthere @eddiejaoude Is this PR ready to be merged? Also, I think @hemantwasthere may commit the above suggested changes.